### DISCUSSION TITLE
Multiple meal planns in one Household

### DISCUSSION THREAD

**Discussion by mhh008** [mealie-recipes/mealie#4157] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Currently it is not possible to have multiple meal plans in one group / household.
This means that it is not possible to have a common recipe base with other family members or friends. Everyone must have their own, otherwise everyone would have to share the meal plan. This is often not possible as not all group members have to live in the same place.

### Please provide a concise description of the feature that would resolve your issue.

Enable the option to create multiple named meal plans in one household. 

### Please consider and list out some caveats or tradeoffs made in your design decision

-

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
> it is not possible to have a common recipe base with other family members or friends

What version are you running? This is one of the primary use-cases of households; one household has one meal plan, another household has another meal plan etc. But recipes are shared between households

If you're on nightly (I assume you are since you mentioned households) we just merged a feature enabling [cross-household recipes](https://github.com/mealie-recipes/mealie/pull/4089) a few days ago, so households can share recipes

**Comment by mhh008** [comment]
v1.12.0 - Language set to German
What do you mean with nightly? I thought group and household are used synonymously.
Do not see a possibility to create something called hosehould.

**Comment by michael-genson** [comment]
Ahhh understood. Nightly is the latest version of Mealie, not guaranteed to be stable. v1.12.0 is the most recent stable version of Mealie.

In nightly, we recently added a new feature called "Households", which does exactly what you're asking. Households share recipes, but maintain their own shopping lists, meal plans, etc. Multiple households belong to a group.

Households are _very_ new, you can read more about the idea here: https://github.com/mealie-recipes/mealie/discussions/2134

**Comment by mhh008** [comment]
Thanks.

**Comment by thedualspace** [comment]
@michael-genson currently the meal plan is assigned to each household (rather than each user). Is it possible for each user to optionally have their own meal plan? 

In a house with many adults, there is a big benefit to having shared shopping lists, but separate meal plans for the week.

I understand the utility for some households to have a singular meal plan, but for many, meal plans for each user makes more sense. This would make a big difference to the practical utility of this app for my household.

**Comment by raseitz1011** [comment]
I may have a unique situation for the need for saving multiple meal plans.  Planning meals for most of the week often feels challenging, so doing this weekly is not a task I can manage without undo stress.  For that reason, I created a rotating menu/meal plan in my old meal planner app.  They were labeled A-H and allowed me to not HAVE to sit down each week and plan out the next 7 days once I set them up.  This is my first week using mealie and I like it a lot except for the single meal plan and I noticed that my plan for Monday disappeared on Tuesday.  Neither adult was feeling well enough to cook Monday so we figured we would just make it later this week.  Problem is now I have all the ingredients but don't remember exactly what I planned to do with them :( If the multiple week function was a feature, I could look and see what I planned for Monday since it was saved.
